[None][doc] Add Qwen3 Next Quick Start Guide

faradawn opened the pull request:
@coderabbitai summary

<!--
Please write the PR title by following this template:

**[JIRA ticket/NVBugs ID/GitHub issue/None][type] Summary**

Valid ticket formats:
  - JIRA ticket: [TRTLLM-1234] or [FOOBAR-123] for other FOOBAR project
  - NVBugs ID: [https://nvbugs/1234567]
  - GitHub issue: [#1234]
  - No ticket: [None]

Valid types (lowercase): [fix], [feat], [doc], [infra], [chore], etc.

Examples:
  - [TRTLLM-1234][feat] Add new feature
  - [https://nvbugs/1234567][fix] Fix some bugs
  - [#1234][doc] Update documentation
  - [None][chore] Minor clean-up

Alternative (faster) way using CodeRabbit AI:

**[JIRA ticket/NVBugs ID/GitHub issue/None] @coderabbitai title**

NOTE: "@coderabbitai title" will be replaced by the title generated by CodeRabbit AI, that includes the "[type]" and title.
For more info, see /.coderabbit.yaml.

-->



## Description

Create a guide for Qwen3 Next usage. Related to this PR: https://github.com/NVIDIA/TensorRT-LLM/pull/7892

## Test Coverage

<!--
Please list clearly what are the relevant test(s) that can safeguard the changes in the PR. This helps us to ensure we have sufficient test coverage for the PR.
-->

## PR Checklist

Please review the following before submitting your PR:
- PR description clearly explains what and why. If using CodeRabbit's summary, please make sure it makes sense.
- PR Follows [TRT-LLM CODING GUIDELINES](https://github.com/NVIDIA/TensorRT-LLM/blob/main/CODING_GUIDELINES.md) to the best of your knowledge.
- Test cases are provided for new code paths (see [test instructions](https://github.com/NVIDIA/TensorRT-LLM/tree/main/tests#1-how-does-the-ci-work))
- Any new dependencies have been scanned for license and vulnerabilities
- [CODEOWNERS](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/CODEOWNERS) updated if ownership changes
- Documentation updated as needed
- The reviewers assigned automatically/manually are appropriate for the PR.


- [x] Please check this after reviewing the above items as appropriate for this PR.

## GitHub Bot Help

`/bot [-h] ['run', 'kill', 'skip', 'reuse-pipeline'] ...`

Provide a user friendly way for developers to interact with a Jenkins server.

Run `/bot [-h|--help]` to print this help message.

See details below for each supported subcommand.

<details>

`run  [--reuse-test (optional)pipeline-id --disable-fail-fast --skip-test --stage-list "A10-PyTorch-1, xxx" --gpu-type "A30, H100_PCIe" --test-backend "pytorch, cpp" --add-multi-gpu-test --only-multi-gpu-test --disable-multi-gpu-test --post-merge --extra-stage "H100_PCIe-TensorRT-Post-Merge-1, xxx" --detailed-log --debug(experimental)]`

Launch build/test pipelines. All previously running jobs will be killed.

`--reuse-test (optional)pipeline-id ` *(OPTIONAL)* : Allow the new pipeline to reuse build artifacts and skip successful test stages from a specified pipeline or the last pipeline if no pipeline-id is indicated. If the Git commit ID has changed, this option will be always ignored. The DEFAULT behavior of the bot is to reuse build artifacts and successful test results from the last pipeline.

`--disable-reuse-test ` *(OPTIONAL)* : Explicitly prevent the pipeline from reusing build artifacts and skipping successful test stages from a previous pipeline. Ensure that all builds and tests are run regardless of previous successes.

`--disable-fail-fast ` *(OPTIONAL)* : Disable fail fast on build/tests/infra failures.

`--skip-test ` *(OPTIONAL)* : Skip all test stages, but still run build stages, package stages and sanity check stages. Note: Does **NOT** update GitHub check status.

`--stage-list "A10-PyTorch-1, xxx"` *(OPTIONAL)* : Only run the specified test stages. Examples: "A10-PyTorch-1, xxx". Note: Does **NOT** update GitHub check status.

`--gpu-type "A30, H100_PCIe"` *(OPTIONAL)* : Only run the test stages on the specified GPU types. Examples: "A30, H100_PCIe". Note: Does **NOT** update GitHub check status.

`--test-backend "pytorch, cpp"` *(OPTIONAL)* : Skip test stages which don't match the specified backends. Only support [pytorch, cpp, tensorrt, triton]. Examples: "pytorch, cpp" (does not run test stages with tensorrt or triton backend). Note: Does **NOT** update GitHub pipeline status.

`--only-multi-gpu-test ` *(OPTIONAL)* : Only run the multi-GPU tests. Note: Does **NOT** update GitHub check status.

`--disable-multi-gpu-test ` *(OPTIONAL)* : Disable the multi-GPU tests. Note: Does **NOT** update GitHub check status.

`--add-multi-gpu-test ` *(OPTIONAL)* : Force run the multi-GPU tests in addition to running L0 pre-merge pipeline.

`--post-merge ` *(OPTIONAL)* : Run the L0 post-merge pipeline instead of the ordinary L0 pre-merge pipeline.

`--extra-stage "H100_PCIe-TensorRT-Post-Merge-1, xxx"` *(OPTIONAL)* : Run the ordinary L0 pre-merge pipeline and specified test stages. Examples: --extra-stage "H100_PCIe-TensorRT-Post-Merge-1, xxx".

`--detailed-log ` *(OPTIONAL)* : Enable flushing out all logs to the Jenkins console. This will significantly increase the log volume and may slow down the job.

`--debug ` *(OPTIONAL)* : **Experimental feature**. Enable access to the CI container for debugging purpose. Note: Specify exactly one stage in the `stage-list` parameter to access the appropriate container environment. Note: Does **NOT** update GitHub check status.

For guidance on mapping tests to stage names, see `docs/source/reference/ci-overview.md`
and the `scripts/test_to_stage_mapping.py` helper.

### kill

`kill  `

Kill all running builds associated with pull request.

### skip

`skip --comment COMMENT `

Skip testing for latest commit on pull request. `--comment "Reason for skipping build/test"` is required. IMPORTANT NOTE: This is dangerous since lack of user care and validation can cause top of tree to break.

### reuse-pipeline

`reuse-pipeline `

Reuse a previous pipeline to validate current commit. This action will also kill all currently running builds associated with the pull request. IMPORTANT NOTE: This is dangerous since lack of user care and validation can cause top of tree to break.

</details>

faradawn committed:
add guide for qwen3 next

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add guide for qwen3 next

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add disable kv cache reuse

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add disable kv cache reuse

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn commented:
Hi @nv-guomingz @byshiue I have tested the serve command. Could you check and let me know of any issue?

chang-l committed:
[https://nvbugs/5542867][fix] Fix the non-determinism issue in the mm_encoder test (#8033)

Signed-off-by: Chang Liu (Enterprise Products) <9713593+chang-l@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add benchmarking results

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add benchmarking results

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add benchmarking results

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add benchmarking results

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add benchmarking results

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add benchmarking results

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

laikhtewari reviewed (commented):
LGTM, but I think we should be protecting these deployment guides with CI testing to prevent new changes from accidentally breaking at least functionality here -- what do you think @juney-nvidia?

pcastonguay committed:
[https://nvbugs/5538098][fix] Checking connection to etcd server in unit test (#8006)

Signed-off-by: Patrice Castonguay <55748270+pcastonguay@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Njuapp committed:
[TRTLLM-6741][fix] Add heuristics for lm head tp size when `enable_lm_head_tp_in_adp=True` (#7891)

Signed-off-by: Cheng Hang <chang@nvidia.com>
Co-authored-by: Yanchao Lu <yanchaol@nvidia.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

dcaox committed:
[None][feat] Return topk logprobs in torch backend (#7976)

Signed-off-by: Cao Dong <87467313+dcaox@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

lucaslie committed:
[#4593][feat] AutoDeploy: Linear Attention Support (SSM + causal_conv + Bamba + Nemotron-H) (#8068)

Signed-off-by: William Zhang <133824995+2ez4bz@users.noreply.github.com>
Signed-off-by: Lucas Liebenwein <11156568+lucaslie@users.noreply.github.com>
Signed-off-by: Chenghao Zhang <211069071+nvchenghaoz@users.noreply.github.com>
Signed-off-by: Frida Hou <201670829+Fridah-nv@users.noreply.github.com>
Signed-off-by: Suyog Gupta <41447211+suyoggupta@users.noreply.github.com>
Co-authored-by: William Zhang <133824995+2ez4bz@users.noreply.github.com>
Co-authored-by: Chenghao Zhang <211069071+nvchenghaoz@users.noreply.github.com>
Co-authored-by: Frida Hou <201670829+Fridah-nv@users.noreply.github.com>
Co-authored-by: Suyog Gupta <41447211+suyoggupta@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

kaiyux committed:
[None] [test] Add MNNVL AlltoAll tests to pre-merge (#7466)

Signed-off-by: Kaiyu Xie <26294424+kaiyux@users.noreply.github.com>
Signed-off-by: Zongfei Jing <20381269+zongfeijing@users.noreply.github.com>
Co-authored-by: Zongfei Jing <20381269+zongfeijing@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

kaiyux committed:
[None] [test] Add MNNVL AlltoAll tests to pre-merge (#7466)

Signed-off-by: Kaiyu Xie <26294424+kaiyux@users.noreply.github.com>
Signed-off-by: Zongfei Jing <20381269+zongfeijing@users.noreply.github.com>
Co-authored-by: Zongfei Jing <20381269+zongfeijing@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

kaiyux committed:
[None] [test] Add MNNVL AlltoAll tests to pre-merge (#7466)

Signed-off-by: Kaiyu Xie <26294424+kaiyux@users.noreply.github.com>
Signed-off-by: Zongfei Jing <20381269+zongfeijing@users.noreply.github.com>
Co-authored-by: Zongfei Jing <20381269+zongfeijing@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

xinhe-nv committed:
[TRTLLM-6239][feat] add test cases into QA test list (#8081)

Signed-off-by: Xin He (SW-GPU) <200704525+xinhe-nv@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

yechank-nvidia committed:
[None][fix] Fix CUDA graph for Qwen2.5-VL (#8047)

Signed-off-by: yechank <161688079+yechank-nvidia@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

nv-guomingz commented on tests/unittest/_torch/multimodal/test_mm_encoder_standalone.py:
why we need to modify this file?

nv-guomingz commented on docs/source/deployment-guide/quick-start-recipe-for-qwen3-next-on-trtllm.md:
````diff
@@ -0,0 +1,291 @@
+
+```
+============ Serving Benchmark Result ============
+Total requests:                          12        
````
Is it the real perf number?

nv-guomingz commented on docs/source/deployment-guide/quick-start-recipe-for-qwen3-next-on-trtllm.md:
```diff
@@ -0,0 +1,291 @@
+
+## Prerequisites
+
+* GPU: NVIDIA Hopper or Blackwell Architecture
```
Did you run this model on Blackwell host yet?

faradawn replied:
Yes, I ran it 4xB200 on our prenyx cluster with the above config.

yiqingy0 committed:
[None][chore] Bump version to 1.2.0rc1 (#8097)

Signed-off-by: Yiqing Yan <yiqingy@nvidia.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
fix the spelling of TensorRT LLM

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
fix the spelling of TensorRT LLM

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
change ngc container version to 1.1.0rc7

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
change ngc container version to 1.1.0rc7

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add docker build command

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add docker build command

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn commented:
Hi @laikhtewari , @ncomly-nvidia the guide is ready and tested on H100 and B200. I have fixed @nv-guomingz 's issue of container v1.1.0rc6 not containing Qwen3.

faradawn committed:
add build and run commands

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add build and run commands

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Tabrizian committed:
[https://nvbugs/5547414][fix] avoid downloading Tiny llama from HF (#8071)

Signed-off-by: Iman Tabrizian <10105175+tabrizian@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

nv-guomingz committed:
[None][chore] Refine qwen3-next implementation. (#8064)

Signed-off-by: nv-guomingz <137257613+nv-guomingz@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add note that more features and optimizations will be released

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add note that more features and optimizations will be released

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
remove mentioning of best performance

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
remove mentioning of best performance

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

ixlmar committed:
[TRTLLM-8269][fix] Revert "do not explicitly pass temperature=0 to select greedy sampling" (#8103)

Signed-off-by: ixlmar <206748156+ixlmar@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

ixlmar committed:
[TRTLLM-8269][fix] Revert "do not explicitly pass temperature=0 to select greedy sampling" (#8103)

Signed-off-by: ixlmar <206748156+ixlmar@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

ixlmar committed:
[TRTLLM-8269][fix] Revert "do not explicitly pass temperature=0 to select greedy sampling" (#8103)

Signed-off-by: ixlmar <206748156+ixlmar@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
remove benchmarking result

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
remove benchmarking result

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
remove benchmarking result

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
remove benchmarking result

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
remove benchmarking result

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
remove benchmarking result

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add blackwell not supported

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add blackwell not supported

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
restore unit tests

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
restore unit tests

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
restore unit tests

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
restore unit tests

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn commented:
1) Added notes on Blackwell is yet supported. 

2) For model feature support matrix, I wrote "no" for all columns except CUDA graph. Can @nv-guomingz and @byshiue help check?

Reference: model huggingface: https://huggingface.co/Qwen/Qwen3-Next-80B-A3B-Thinking

faradawn committed:
add to model feature support matrix

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
add to model feature support matrix

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

brb-nv committed:
[None][chore] Waive failing MNNVL alltoall multi-gpu test (#8106)

Signed-off-by: Balaram Buddharaju <169953907+brb-nv@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

brb-nv committed:
[None][chore] Waive failing MNNVL alltoall multi-gpu test (#8106)

Signed-off-by: Balaram Buddharaju <169953907+brb-nv@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

brb-nv committed:
[None][chore] Waive failing MNNVL alltoall multi-gpu test (#8106)

Signed-off-by: Balaram Buddharaju <169953907+brb-nv@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

peaceh-nv committed:
[None][fix] : Fix OOM issue when dp padding is enabled (#8052)

Signed-off-by: peaceh <103117813+peaceh-nv@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

yechank-nvidia committed:
[None][doc] Add more description on EXAONE usage (#8089)

Signed-off-by: yechank <161688079+yechank-nvidia@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

EmmaQiaoCh committed:
[None][infra] Skip failed tests in post-merge for main (#8102)

Signed-off-by: qqiao <qqiao@nvidia.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

pcastonguay committed:
[https://nvbugs/5434320][fix] fix: Unwaiving disagg pp tests (#8069)

Signed-off-by: Patrice Castonguay <55748270+pcastonguay@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

sychen52 committed:
[OMNIML-2336][feat] add W4A8 NVFP4 FP8 fused moe (#7968)

Signed-off-by: Shiyang Chen <shiychen@nvidia.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Funatiq committed:
[None][test] Add accuracy test for Qwen3Next model

- Updated L0_Test.groovy to include auto_trigger for Qwen3Next.
- Added TestQwen3NextThinking class in test_llm_api_pytorch.py for accuracy testing.
- Updated l0_dgx_h100.yml to include new test case for Qwen3Next under specific conditions.

Signed-off-by: Robin Kobus <19427718+Funatiq@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Funatiq committed:
[None][test] Add accuracy test for Qwen3Next model

- Updated L0_Test.groovy to include auto_trigger for Qwen3Next.
- Added TestQwen3NextThinking class in test_llm_api_pytorch.py for accuracy testing.
- Updated l0_dgx_h100.yml to include new test case for Qwen3Next under specific conditions.

Signed-off-by: Robin Kobus <19427718+Funatiq@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

greg-kwasniewski1 committed:
[TRTLLM-6342][bug] Fix shape propagation after TP sharding (#7912)

Signed-off-by: greg-kwasniewski1 <213329731+greg-kwasniewski1@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

yibinl-nvidia committed:
[TRTLLM-8031][feat] Add chunked return_generation_logits logic (#7831)

Signed-off-by: Yibin Li <109242046+yibinl-nvidia@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
change parameters to match CI test in #8111

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

faradawn committed:
change parameters to match CI test in #8111

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Fridah-nv committed:
[#5860][feat] Add ModelOPT INT4 awq fake quant support in AutoDeploy (#7770)

Signed-off-by: Frida Hou <201670829+Fridah-nv@users.noreply.github.com>
Signed-off-by: Fridah-nv <201670829+Fridah-nv@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

bo-nv committed:
[None][fix] fix patchelf version issue (#8112)

Signed-off-by: Bo Deng <deemod@nvidia.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Commit:
[None][feat] Draft: Save state first pass (#7012)

Signed-off-by: Izzy Putterman <iputterman@nvidia.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

brb-nv committed:
[TRTLLM-7733][feat] Executor changes to support helix parallelism (#7972)

Signed-off-by: Balaram Buddharaju <169953907+brb-nv@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

chang-l committed:
[https://nvbugs/5549081][fix] Fix device id assignment for some vision models (#8070)

Signed-off-by: Chang Liu (Enterprise Products) <9713593+chang-l@users.noreply.github.com>
Signed-off-by: Chang Liu <9713593+chang-l@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

MrGeva committed:
[#7588][feat] lock gpu clocks in test_perf.py to reliably detect perf regressions (#8099)

Signed-off-by: Eran Geva <19514940+MrGeva@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

ixlmar committed:
[TRTLLM-8269][test] do not explicitly pass temperature=0 to select greedy sampling (#8110)

Signed-off-by: ixlmar <206748156+ixlmar@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

ixlmar committed:
[TRTLLM-8269][test] do not explicitly pass temperature=0 to select greedy sampling (#8110)

Signed-off-by: ixlmar <206748156+ixlmar@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

ixlmar committed:
[TRTLLM-8269][test] do not explicitly pass temperature=0 to select greedy sampling (#8110)

Signed-off-by: ixlmar <206748156+ixlmar@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Funatiq committed:
Revert "restore unit tests"

This reverts commit 196cf929bc86095439b9709cd23e9d0edd071812.

Signed-off-by: Robin Kobus <19427718+Funatiq@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Funatiq committed:
Revert "restore unit tests"

This reverts commit 196cf929bc86095439b9709cd23e9d0edd071812.

Signed-off-by: Robin Kobus <19427718+Funatiq@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Funatiq committed:
Revert "restore unit tests"

This reverts commit 196cf929bc86095439b9709cd23e9d0edd071812.

Signed-off-by: Robin Kobus <19427718+Funatiq@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Funatiq committed:
Revert "restore unit tests"

This reverts commit 196cf929bc86095439b9709cd23e9d0edd071812.

Signed-off-by: Robin Kobus <19427718+Funatiq@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

hchings committed:
[https://nvbugs/5556020][chore] waive test_eagle3 (#8119)

Signed-off-by: Erin Ho <14718778+hchings@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

tensorrt-cicd commented:
[PR_Github #20517](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/20517/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #15476](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/15476/) completed with status: 'FAILURE'

tensorrt-cicd commented:
[PR_Github #20526](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/20526/) [ run ] triggered by Bot

tensorrt-cicd commented:
[PR_Github #20532](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/20532/) [ run ] triggered by Bot

yifeizhang-c committed:
[TRTLLM-6589][feat] Support CUDA graph for DeepEP (#7514)

Signed-off-by: Yifei Zhang <219273404+yifeizhang-c@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

dongfengy committed:
[TRTLLM-7775][feat] Integrate tinygemm2 for gpt-oss (#7916)

Signed-off-by: Dongfeng Yu <dongfengy@nvidia.com>
Signed-off-by: dongfengy <99041270+dongfengy@users.noreply.github.com>
Co-authored-by: Jin Li <59594262+liji-nv@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

pcastonguay committed:
[None][feat] Support for cancelling requests with disaggregation (#8114)

Signed-off-by: Shunkang <182541032+Shunkangz@users.noreply.github.co>
Signed-off-by: Patrice Castonguay <55748270+pcastonguay@users.noreply.github.com>
Co-authored-by: Shunkang <182541032+Shunkangz@users.noreply.github.co>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

tensorrt-cicd commented:
[PR_Github #20532](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/20532/) [ run ] completed with state `SUCCESS`
[/LLM/main/L0_MergeRequest_PR pipeline #15489](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/15489/) completed with status: 'FAILURE'

dcampora committed:
[None][fix] Fix access to new tokens in sampler. (#7958)

Signed-off-by: Daniel Campora <961215+dcampora@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

pcastonguay committed:
[None][chore] Adding install_tensorrt.sh script to pip wheel (#8116)

Signed-off-by: Patrice Castonguay <55748270+pcastonguay@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

MrGeva committed:
[#7588][fix] fixed the kv cache size parsing in test_perf.py AD backend (#8092)

Signed-off-by: Eran Geva <19514940+MrGeva@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

greg-kwasniewski1 committed:
[TRTLLM-6342][bug] Patched incorrect starcoder tp config (#8118)

Signed-off-by: greg-kwasniewski1 <213329731+greg-kwasniewski1@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

nv-yilinf committed:
[None][feat] perf_metrics endpoint functionality improvement (#8005)

Signed-off-by: Yilin Fan <206948969+nv-yilinf@users.noreply.github.com>
Signed-off-by: nv-yilinf <206948969+nv-yilinf@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

nekorobov committed:
[None][feat] Update TRT-LLM Gen MoE kernels (#7970)

Signed-off-by: Nikita Korobov <14355239+nekorobov@users.noreply.github.com>
Signed-off-by: Faradawn Yang <faradawny@gmail.com>

hchings committed:
[https://nvbugs/5548098][fix] Fix flakey unit test for dynamic spec d… (#8129)

Signed-off-by: Faradawn Yang <faradawny@gmail.com>

Funatiq commented:
Sorry, I rebased to ToT main and that broke DCO again. Could please rebase onto main again with your fix?

mikeiovine committed:
[None][fix] Fix MTP 2-model (#8115)

Signed-off-by: Mike Iovine <6158008+mikeiovine@users.noreply.github.com>
Signed-off-by: Mike Iovine <miovine@nvidia.com>

moraxu committed:
[TRTLLM-6496][feat] Add LoRa Torch tests for the latest NIM model list (#6806)

Signed-off-by: Michal Guzek <mguzek@nvidia.com>

lucaslie committed:
[None][feat] AutoDeploy: Nemotron-H accuracy test (#8133)

Signed-off-by: Lucas Liebenwein <11156568+lucaslie@users.noreply.github.com>

lucaslie committed:
[None][feat] AutoDeploy: graph/module inputs with kwargs instead of args (#8137)

Signed-off-by: Lucas Liebenwein <11156568+lucaslie@users.noreply.github.com>

joyang-nv committed:
[TRTLLM-7349][feat] Adding new orchestrator type --  ray (#7520)

Signed-off-by: Erin Ho <14718778+hchings@users.noreply.github.com>
Co-authored-by: Yuan Tong <13075180+tongyuantongyu@users.noreply.github.com>
Co-authored-by: Erin Ho <14718778+hchings@users.noreply.github.com>

Fridah-nv committed:
[None][autodeploy] small refactors on attention matching (#8079)

Signed-off-by: Frida Hou <201670829+Fridah-nv@users.noreply.github.com>
Signed-off-by: Fridah-nv <201670829+Fridah-nv@users.noreply.github.com>

Fridah-nv committed:
[#5255][autodeploy] Update FuseAllreduceResidualRMSNorm to use pattern matcher utility; remove fuse_collective (#7545)

Signed-off-by: Frida Hou <201670829+Fridah-nv@users.noreply.github.com>
Signed-off-by: Fridah-nv <201670829+Fridah-nv@users.noreply.github.com>

Superjomn committed:
[TRTLLM-8189][chore] enhance GenerationExecutor with RPC (part1) (#5543)

Signed-off-by: Yan Chunwei <328693+Superjomn@users.noreply.github.com>
Signed-off-by: chunweiy <chunweiy@nvidia.com>
Signed-off-by: Superjomn <328693+Superjomn@users.noreply.github.com>
Signed-off-by: chunweiy <328693+Superjomn@users.noreply.github.com>

amitz-nv committed:
[https://nvbugs/5521949][fix] Re-enable test_bielik_11b_v2_2_instruct_multi_lora, fix its API use with pytorch flow LoRA (#8146)

Signed-off-by: Amit Zuker <203509407+amitz-nv@users.noreply.github.com>

pcastonguay committed:
[None][fix] Adding docker folder to Dockerfile (#8138)

Signed-off-by: Patrice Castonguay <55748270+pcastonguay@users.noreply.github.com>

Superjomn committed:
[None][chore] fix llmargs conflict (#8152)

Signed-off-by: Yan Chunwei <328693+Superjomn@users.noreply.github.com>

ixlmar committed:
[TRTLLM-8413][chore] resolve sampling defaults in OpenAI API backend (#8121)

Signed-off-by: ixlmar <206748156+ixlmar@users.noreply.github.com>

lucaslie committed:
[None][chore] AutoDeploy: clean up accuracy test configs (#8134)

Signed-off-by: Lucas Liebenwein <11156568+lucaslie@users.noreply.github.com>

Commit:
[None][fix] Eagle: Attention DP (#7939)

Signed-off-by: Izzy Putterman <iputterman@nvidia.com>

farazkh80 committed:
[None][feat] GPT-OSS Sm120/Sm121 Support (#7937)

Signed-off-by: Perkz Zheng <67892460+PerkzZheng@users.noreply.github.com>
Signed-off-by: list <58580514+farazkh80@users.noreply.github.com>
Signed-off-by: Vincent Huang <vincenth@nvidia.com>
Co-authored-by: Perkz Zheng <67892460+PerkzZheng@users.noreply.github.com>
Co-authored-by: Vincent Huang <vincenth@nvidia.com>

karljang committed:
[None][chore] Increase operations-per-run to 1000 for stale action (#8162)

Signed-off-by: Kanghwan Jang <861393+karljang@users.noreply.github.com>

VALLIS-NERIA committed:
[None] [test] Add B300 cases to CI (#8056)

Signed-off-by: Xiwen Yu <13230610+VALLIS-NERIA@users.noreply.github.com>

Funatiq commented:
> @nv-guomingz @Funatiq Hi is there any issue with this PR? Thanks

I'm not sure how you're adding the sign-off, but it seems that the branch includes a lot of commits from main after that. And because of that the PR shows many file changes and added many reviewers (code owners).

EmmaQiaoCh committed:
[None][infra] Skip failed cases for main (#8176)

Signed-off-by: qqiao <qqiao@nvidia.com>

mikeiovine committed:
[None][fix] Fix MTP illegal memory access (#8161)

Signed-off-by: Mike Iovine <6158008+mikeiovine@users.noreply.github.com>

faradawn committed:
Merge branch qwen3-next-guide of github.com:faradawn/TensorRT-LLM into qwen3-next-guide

Signed-off-by: Faradawn Yang <faradawny@gmail.com>
